fakes/bar.go

```diff
@@ -0,0 +1,17 @@
+package fakes
```

Comment:
With the interfaces exported both this and the fake ranger could just be generated with `counterfeiter`.